src/ConfirmDeleteAllModal.js

```diff
@@ -46,6 +48,16 @@ class ConfirmDeleteAllModal extends React.Component<ConfirmDeleteAllModalProps,
             </Modal>
         );
     }
+
+    componentDidUpdate (prevProps: ConfirmDeleteAllModalProps) {
+        if (prevProps.show !== this.props.show && this.props.show) {
```

Comment:
We use this pattern often, and we have a general version of this in App.js.  Longer term I think we should move that function to Utils.js and replace all usages.  I will create a new ticket for that cleanup, but it would be nice to leave a TODO here.